kafka_utils/util/zookeeper.py

```diff
@@ -39,6 +39,15 @@ def load_json(data_str):
         return json.loads(data_str)
 
 
+def dump_json(obj):
```

Comment:
What about we move this function in a separate module so we do not need to re-implement in the test?